package-lock.json

```
@@ -1,15 +1,15 @@
 {
   "name": "@umbraco/playwright-testhelpers",
-  "version": "2.0.0-beta.23",
+  "version": "2.0.0-beta.24",
```

```suggestion
  "version": "2.0.0-beta.23",
```
since the version 2.0.0-beta.23 is not released yet.